lib/rspec/core/formatters/bisect_progress_formatter.rb

```diff
@@ -26,17 +28,40 @@ def bisect_original_run_complete(notification)
           output.puts "Starting bisect with #{failures} and #{non_failures}."
         end
 
+        def bisect_dependency_check_started(_notification)
+          output.print "Checking that failures are order-dependent.."
+        end
+
+        def bisect_dependency_check_finished(notification)
+          is_part = notification.dependent_failures ? "is" : "is not"
```

Comment:
Seeing this conditional makes me think that maybe it would be better to have separate notifications for the two cases, e.g. `bisect_dependency_check_succeeded` and `bisect_dependency_check_failed` or similar.  Thoughts?